events/class_tests/meeting_tests.py

```diff
@@ -2,10 +2,10 @@
 from datetime import datetime
 from django.utils import timezone
 from users.tests import user0  # noqa: F401
-from .event_tests import new_event  # noqa: F811 ,F401
+from events.tests import new_event  # noqa: F811 ,F401
 from django.core.exceptions import ValidationError
 from events.models import OptionalMeetingDates, EventParticipant, Event
-from .participant_test import event_participant_not_creator  # noqa:F811, F401
+from events.tests import event_participant_not_creator  # noqa:F811, F401
```

Comment:
you already have this "from" in line # 5 so you can do it in one line like:
```suggestion
from events.tests import new_event, event_participant_not_creator  # noqa:F811, F401
```

Reply:
done 😄